Nodejs/Product/Nodejs/Workspace/ContextMenuProvider.cs

```diff
@@ -67,14 +69,17 @@ public NpmCommandHandler(OutputPaneWrapper outputPane)
 
             public int Exec(List<WorkspaceVisualNodeBase> selection, Guid pguidCmdGroup, uint nCmdID, uint nCmdexecopt, IntPtr pvaIn, IntPtr pvaOut)
             {
-                var node = selection.FirstOrDefault();
```

Comment:
> var node = selection.FirstOrDefault(); [](start = 15, length = 39)

What was wrong with the old version?

Reply:
Nothing, but this matches the QueryStatus method